build/devops_gated.yml

```diff
@@ -94,7 +94,7 @@ jobs:
     - vstest
 
   steps:
-  - task: BatchScript@1
+  - task: BatchScript@1z
```

Comment:
>z [](start = 23, length = 1)

seems like a random z was accidentally inserted here

Reply:
whoops

Reply:
:)

---
In reply to: [434183719](https://github.com/Azure/azure-c-util/pull/46#discussion_r434183719) [](ancestors = 434183719)